app/controllers/v2/ManageUserController.scala

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.v2
+
+import cats.syntax.traverse._
+import com.google.inject.Inject
+import config.AppConfig
+import controllers.RequestActions
+import models._
+import models.forms.v2.UserEditTeamForm
+import models.request.AuthenticatedRequest
+import models.viewmodels.{ManagerToolsUsersTab, SubNavigationTab, _}
+import play.api.Logging
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import service.{CasesService, EventsService, UserService}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ManageUserController @Inject() (
+  verify: RequestActions,
+  casesService: CasesService,
+  eventsService: EventsService,
+  userService: UserService,
+  mcc: MessagesControllerComponents,
+  val viewUser: views.html.partials.users.view_user,
+  val user_team_edit: views.html.partials.users.user_team_edit
+)(
+  implicit val appConfig: AppConfig,
+  ec: ExecutionContext
+) extends FrontendController(mcc)
+    with I18nSupport
+    with Logging {
+
+  private val userEditTeamForm = UserEditTeamForm.editTeamsForm
+
+  private def getReferralEvents(
```

Comment:
might need getting up to date with DIT-2692 as the methods to get referral events were not needed and got removed

Reply:
Perfect, thanks! We have removed these two methods 👍